### DISCUSSION TITLE
The chosen alias is already in use.

### DISCUSSION THREAD

**Discussion by Deanstirrat** [vercel/community#1793] [Help]
### Summary

I'm getting this error even though I never specified an alias and cannot find any alias by that name via the cli. I built the project using vercel.com and connecting a repo. Can't find any other info about this. The name of the alias is completely different that my app name and I'm not sure where it came from.

**Comment by amyegan** [comment]
Hi Dean. Are you able to share the error message or a minimal reproduction so we can get a better look at the problem?

**Reply by Deanstirrat** [reply]
<img width="1248" alt="Screenshot 2023-03-15 at 10 55 43 AM" src="https://user-images.githubusercontent.com/37011725/225400494-c6572668-514c-4fd7-9cbd-268bf1b910ff.png">
